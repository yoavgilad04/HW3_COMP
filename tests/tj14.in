//Earliest fail degenerate case

override int foo(int bar){return 1;}

override
int
foo
(
int
bar
)
{
return 1;
}