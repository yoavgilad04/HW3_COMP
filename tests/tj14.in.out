---end scope---
bar INT -1
line 11: identifier foo is already defined
